### PR TITLE
rgw: get or set realm zonegroup zone need check user's caps

### PR DESCRIPTION
**Pull request by joke-lee**
fix https://tracker.ceph.com/issues/37352
Signed-off-by: yuliyang <yuliyang@cmss.chinamobile.com>

### CODE REVIEW

**Comment by joke-lee** [comment]
hi @cbodley , would you mind take a  review? thanks

File: src/rgw/rgw_rest_config.h [hunk 1]
```diff
@@ -24,8 +24,11 @@ class RGWOp_ZoneGroupMap_Get : public RGWRESTOp {
   explicit RGWOp_ZoneGroupMap_Get(bool _old_format):old_format(_old_format) {}
   ~RGWOp_ZoneGroupMap_Get() override {}
 
+  int check_caps(RGWUserCaps& caps) override {
+    return caps.check_cap("admin", RGW_CAP_READ);
```
Comment (cbodley) [inline src/rgw/rgw_rest_config.h]: let's just use the "zone" cap for these?

**Comment by cbodley** [comment]
can you please create a track issue so we can backport this?

**Comment by joke-lee** [comment]
jenkins test this please

**Comment by joke-lee** [comment]
hi, @cbodley , tracker issue have created

**Comment by joke-lee** [comment]
jenkins test this please

**Comment by cbodley** [comment]
hi @joke-lee, i'm seeing multisite test failures from this where `period update --commit` fails with 403 in 
`test_zonegroup_remove()`, because the removed zone doesn't contain the admin credentials. i added a commit https://github.com/cbodley/ceph/commit/394100fb2c419c64bf176b1b57b79fe1fdd45ebc that fixes this - could you please pull it into your pr?

**Pull request merged**